src/App.js

```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import { connect } from 'react-redux';
+import React from 'react';
+import { Actions } from './redux/actions';
 
-function App() {
+const App = ({ isLoggedIn, loginEpic }) => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>is Logged In: {isLoggedIn.toString()}</h1>
+      <button onClick={loginEpic} type="button">
+        Log In
+      </button>
     </div>
   );
-}
+};
 
-export default App;
+const mapStateToProps = (state) => ({
+  isLoggedIn: state.isLoggedIn,
```

Comment:
We will be using the (useSelector and useDispatch) hook for functional component. So avoid connect in functional components..